### ISSUE TITLE
Quests

### ISSUE THREAD

**Issue by Jeremy9600** [GLServers/GLD#47]
Kerrus' Idea:

Have quests for new players starting off, and, possibly more experienced players.

Quests could range from:

_Mine one stack of diamonds_

_Chop down a tree_

And for more experienced players

_Gain 800 levels in any non-combat related McMMO skill_

_Gain one million lightstones/emeralds_

Rewards could also be given out once a player has completed their achievement/quest.

I'm sure it's possible!

**Comment by Joannou1** [comment]
Yes I would like to see custom achievements or simple quests be a thing.

**Comment by Joannou1** [comment]
Renamed to Quests, see Achievements at #21 
Set to launch cause we need to figure out how everything will be set up first before we make quests/achievements.
